ResourceStatusSystem/Command/GOCDBSyncCommand.py

```diff
@@ -0,0 +1,105 @@
+'''
+  GOCDBSyncCommand module
+
+  This command updates the downtime dates from the DowntimeCache table in case they changed
+  after being fetched from GOCDB. In other words, it ensures that all the downtime dates in
+  the database are current.
+
+'''
+
+import errno
+import requests
+import xml.dom.minidom as minidom
+from DIRAC                                                      import S_OK, S_ERROR
+from DIRAC.Core.LCG.GOCDBClient                                 import GOCDBClient
+from DIRAC.Core.LCG.GOCDBClient                                 import _parseSingleElement
+from DIRAC.ResourceStatusSystem.Command.Command                 import Command
+from DIRAC.ResourceStatusSystem.Client.ResourceManagementClient import ResourceManagementClient
+
+__RCSID__ = '$Id:  $'
+
+class CheckStatusCommand( Command ):
+
+  def __init__( self, args = None, clients = None ):
+
+    super( CheckStatusCommand, self ).__init__( args, clients )
+
+    if 'GOCDBClient' in self.apis:
+      self.gClient = self.apis[ 'GOCDBClient' ]
+    else:
+      self.gClient = GOCDBClient()
+
+    if 'ResourceManagementClient' in self.apis:
+      self.rmClient = self.apis[ 'ResourceManagementClient' ]
+    else:
+      self.rmClient = ResourceManagementClient()
+
+  def doNew( self, masterParams = None ):
+    """
+    Gets the downtime IDs and dates of a given hostname from the local database and compares the results
+    with the remote database of GOCDB. If the downtime dates have been changed it updates the local database.
+    """
+
+    if masterParams:
+      hostname = masterParams
+    else:
+      return S_ERROR(errno.EINVAL, 'masterParams is not provided')
+
+    result = self.rmClient.selectDowntimeCache( name = hostname )
+    if not result[ 'OK' ]:
+      return result
+
+    for downtimes in result['Value']:
+
+      localDBdict = { 'DowntimeID': downtimes[1],
+                      'FORMATED_START_DATE': downtimes[0].strftime('%Y-%m-%d %H:%M'),
+                      'FORMATED_END_DATE': downtimes[3].strftime('%Y-%m-%d %H:%M') }
+
+      r = requests.get('https://goc.egi.eu/gocdbpi_v4/public/?method=get_downtime&topentity=' + hostname)
```

Comment:
actually, you should add
```python
res.raise_for_status()
```
after the get. This will also ensure that the return value is 200.